siliconcompiler/flows/fpga_nextpnr_flow.py

```diff
@@ -0,0 +1,80 @@
+import siliconcompiler
+
+from siliconcompiler.flows._common import setup_frontend
+
+from siliconcompiler.tools.yosys import syn_fpga
+from siliconcompiler.tools.nextpnr import apr as nextpnr_apr
+
+
+############################################################################
+# DOCS
+############################################################################
+def make_docs(chip):
+    return setup(chip)
+
+
+############################################################################
+# Flowgraph Setup
+############################################################################
+def setup(chip, flowname='fpga_nextpnr_flow'):
+    '''
+    A configurable FPGA compilation flow.
+
+    The 'fpgaflow' module is a configurable FPGA flow with support for
+    open source and commercial tool flows.
```

Comment:
which commercial tools? I would cleanup this docstring

Reply:
Comment deleted.